### DISCUSSION TITLE
undefined ls-remote -h -t ssh://git@github.com/uNetworking/uWebSockets.js.git

### DISCUSSION THREAD

**Discussion by abelardolg** [uNetworking/uWebSockets.js#139] [Q&A] [answered]
Hi there,

NPM version: 6.9.0
Node: 10.15.3 LTS
Windows: 10
uWebSockets.js version: latest.

When I run 

> npm install uNetworking/uWebSockets.js#v15.10.0

the showed message is 

> undefined ls-remote -h -t ssh://git@github.com/uNetworking/uWebSockets.js.git

What's wrong?

Any help would be welcome.

Regards.

**Comment by ghost** [comment]
I don't know. It works for me and others so I cannot confirm this issue. Just tested it to be extra certain and it worked.

**Comment by dbl0null** [comment]
> Hi there,
> 
> NPM version: 6.9.0
> Node: 10.15.3 LTS
> Windows: 10
> uWebSockets.js version: latest.
> 
> When I run
> 
> > npm install uNetworking/uWebSockets.js#v15.10.0
> 
> the showed message is
> 
> > undefined ls-remote -h -t ssh://git@github.com/uNetworking/uWebSockets.js.git
> 
> What's wrong?
> 
> Any help would be welcome.
> 
> Regards.

Just had similar issue on fresh vps instance - installing **git** totally helped. :)

**Comment by jorioux** [comment, marked as answer]
`git` must be in the path environment. After correctly installing it, the issue is solved.

**Reply by eziokenway061** [reply]
thx bro， i need restart my win11 system , then the path environment will be effect ,and solve this issue
